dcc-storage-server/src/main/java/org/icgc/dcc/storage/server/config/SecurityConfig.java

```diff
@@ -115,10 +115,12 @@ private void configureAuthorization(HttpSecurity http) throws Exception {
     http
       .authorizeRequests()
       .antMatchers("/health").permitAll()
-      .antMatchers("/upload/**")
-      .access("#oauth2.hasScope('" + uploadScope + "')")
-      .antMatchers("/download/**")
-      .access("#oauth2.hasScope('" + downloadScope + "')")
+      .antMatchers("/upload/**").permitAll()
+//      .antMatchers("/upload/**")
```

Comment:
Lets remove these and add a comment stating that we are using method level security and link to the new class to look at.